packages/types/lib/linkedProfile/index.ts

```diff
@@ -0,0 +1,14 @@
+export interface LinkedProfile {
+    readonly id: number;
+    readonly profileId: string;
+    readonly accountId: number;
+    readonly environmentId: number;
+    readonly email: string;
+    readonly displayName?: string | null;
+    readonly organization?: {
+        readonly organizationId: string;
+        readonly displayName?: string | null;
+    } | null;
+    readonly createdAt: Date;
+    readonly updatedAt: Date | null;
+}
```

Comment:
nb: readonly is nice when you don't manipulate but can be annoying as lib user.
```ts
const lp: Partial<LinkedProfile> = {
    accountId: 1
}

// Not allowed bc it's readonly
lp.email = 'example@dev';
```

Not saying it's an issue in that case but just fyi

Reply:
noted